sdk/python/kfp/_client.py

```diff
@@ -125,15 +125,18 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client
     config = self._load_config(host, client_id, namespace, other_client_id, other_client_secret, existing_token, proxy, ssl_ca_cert)
     # Save the loaded API client configuration, as a reference if update is
     # needed.
+    self._load_context_setting_or_default()
     self._existing_config = config
-    api_client = kfp_server_api.api_client.ApiClient(config, cookie=cookies)
+    api_client = kfp_server_api.api_client.ApiClient(config,
+        cookie=self._context_setting.get('cookie'),
```

Comment:
Is it secure to store cookies in a file?

Reply:
Agree this may not be the best long term solution, but I think we can leave the choice to users. If they are okay with putting cookies in a file, we can allow them to do so.